crates/teloxide-macros/src/command_attr.rs

```diff
@@ -47,8 +48,30 @@ impl CommandAttrs {
     pub fn from_attributes(attributes: &[Attribute]) -> Result<Self> {
         use CommandAttrKind::*;
 
+        let docs = attributes.iter().filter_map(|attr| {
```

Comment:
I really don't like this workaround and `is_command_description`/`parse_command_description`....

Reply:
I think the actual solution is to change `fold_attrs` so that it can work with doc comments too...

Maybe it should turn

```
#[doc = "blah"]
#[command(description = "x", hide)]
```

Into (conceptually) a stream of
- `doc = "blah"`
- `command.description = "x"`
- `command.hide`

If you don't want to work on that (and listen to my nitpicks afterwards 😅 ), I could make this refactor